tests/ignite/contrib/metrics/regression/test_median_absolute_error.py

```diff
@@ -158,13 +158,15 @@ def _test(n_epochs, metric_device):
         metric_device = torch.device(metric_device)
         n_iters = 80
         size = 105
-        y_true = torch.rand(size=(size,)).to(device)
-        y_preds = torch.rand(size=(size,)).to(device)
+
+        offset = n_iters * size
+        y_true = torch.rand(size=(offset * idist.get_world_size(),)).to(device)
+        y_preds = torch.rand(size=(offset * idist.get_world_size(),)).to(device)
```

Comment:
@sayantan1410 either revert these modifications or make them coherent to others and rename the PR.

Reply:
Yeah, making them coherent with others, also some tests are failing, fixing them as well

Reply:
Have you updated others ? Do not forget to rename the PR title to reflect that you change not only ` test_precision_recall_curve.py`

Reply:
Yes, in the other tests this has been changed and I have changed the PR title as well.

Reply:
Why don't you add
```
        rank = idist.get_rank()
        torch.manual_seed(rank)
```
here and other regression files ?

Reply:
I added this to precision_recall_curve and was waiting for all the tests to pass for it. Now I will add it to the other files as well.